heimlig/src/integration/raw_jobs.rs

```diff
@@ -9,7 +9,17 @@ type BoolRaw = u32; // 0 == false, 1 == true
 
 // TODO: replace with core::mem::variant_count::<RequestRaw>(); once it is stable
 // https://github.com/rust-lang/rust/issues/73662
-const REQUEST_RAW_VARIANTS: u8 = 9;
+const REQUEST_RAW_VARIANTS: u8 = 25;
+const RESPONSE_RAW_VARIANTS: u8 = 18;
```

Comment:
Does it make sense to try `EnumCount` before `variant_count` is stable?
https://docs.rs/strum/0.25.0/strum/trait.EnumCount.html
https://docs.rs/strum_macros/0.25.3/strum_macros/derive.EnumCount.html

Reply:
Thanks. I did not know that crate.